### ISSUE TITLE
Package request: Haveno

### ISSUE THREAD

**Issue by shortwavesurfer2009** [NixOS/nixpkgs#293842]
**Project description**
Haveno is a Decentralized exchange similar to bisq that uses Monero

**Metadata**

* homepage URL: https://haveno.exchange/
* source URL: https://github.com/haveno-dex/haveno
* license: mit, bsd, gpl2+ , ...
* platforms: unix, linux, darwin, ... Linux

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by nahuhh** [comment]
Is there a bounty?

**Comment by shortwavesurfer2009** [comment]
@nahuhh no

**Comment by nahuhh** [comment]
> @nahuhh no

Let me rephrase:
do you think this can/will be completed soon?
if not, do you mind asking #haveno for a bounty

**Comment by shortwavesurfer2009** [comment]
@nahuhh This is my first time submitting anything here, so I do not know what a normal time frame is.

**Comment by nahuhh** [comment]
Ok let me rerephrase:

have you asked #haveno at all? Takes 2mins

**Comment by shortwavesurfer2009** [comment]
@nahuhh I have brought it up in passing, but have not made any kind of formal request to them.

**Comment by Kreyren** [comment]
bump bump bump it was just released

**Comment by nahuhh** [comment]
related https://github.com/milahu/nixpkgs/issues/68

**Comment by Kreyren** [comment]
I wouldn't trust the milahu's implementation for reasons explained. I am working on packaging it, but currently running into issues.
